doc/source/tutorials/pidloops.rst

```diff
@@ -3,6 +3,22 @@
 PID Loops in kOS
 ================
 
+.. versionadded:: 0.18.1
+
+    Note, this is an older tutorial.  As of 
+    kOS version 0.18.1 and up, a new :struct:`pidloop`
+    feature was added to kOS to allow you to use a built-in PID
+    controller that executes very quickly in the kOS "hardware"
+    rather than in your script code.  You can use it to perform
+    the work described in detail on this page.  However, this
+    tutorial is still quite important because it walks you through
+    how a PID controller works and what it's really doing under the
+    hood.  It's probably a good idea to use the built-in
+    :struct:`pidloop` instead of the program shown here, once you
+    understand the topic this page describes.  However, it's also
+    a good idea to have a read through this page to get an 
+    understanding of what that built-in feature is really doing.
+
```

Comment:
Im glad we didnt simply get rid of the old tutorial. it is still useful for understanding PID  concepts